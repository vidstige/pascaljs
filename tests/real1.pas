program real1;

var
    x: Real;
    y: Real;
begin
    x := 7;
    y := x / 2;
    WriteLn('y: ', y);
end.