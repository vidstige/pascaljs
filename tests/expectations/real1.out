y: 3.5
